iOS/Core/PixelEvent.swift

```diff
@@ -2461,6 +2467,12 @@ extension Pixel.Event {
         case .aiChatMetricSentPromptOngoingChat: return "m_aichat_sent_prompt_ongoing_chat"
         case .aiChatInternalSwitchBarDisplayed: return "m_aichat_internal_switch_bar_displayed"
 
+        // MARK: Privacy Pro Toolbar Button
+        case .privacyProToolbarButtonShown: return "m_privacy_pro_toolbar_button_shown"
+        case .privacyProToolbarButtonPopoverShown: return "m_privacy_pro_toolbar_button_popover_shown"
+        case .privacyProToolbarButtonPopoverNoThanksClicked: return "m_privacy_pro_toolbar_button_popover_no_thanks_clicked"
+        case .privacyProToolbarButtonPopoverTryForFreeClicked: return "m_privacy_pro_toolbar_button_popover_try_for_free_clicked"
+
```

Comment:
Same as above, these should be macOS pixels. Also, the naming for macOS pixels is different - pixels start with `m_mac…`. This prefix can be explicitly defined in the pixel string you are adding, or it can be omitted in which case it will automatically be added by `PixelKit`. A recent reference for adding macOS pixels is here - https://github.com/duckduckgo/apple-browsers/pull/1332/files.